site/pages/create-disruption.tsx

```diff
@@ -1,5 +1,211 @@
+import DateSelector from "components/DateSelector";
+import TimeSelector from "components/TimeSelector";
+import { BaseLayout } from "components/layout/Layout";
+
+const title = "Create Disruptions";
+const description = "Create Disruptions page for the Create Transport Disruptions Service";
+
+enum Reason {
+    roadWorks = "roadWorks",
+    vandalism = "vandalism",
+    routeDiversion = "routeDiversion",
+    specialEvent = "specialEvent",
+}
+
+const renderOption = (text: string) => {
```

Comment:
If we do the above, then we dont need to use this function

Reply:
Removed this method as the enum values have user friendly names